charts/stash/templates/validating-webhook.yaml

```diff
@@ -15,7 +15,7 @@ webhooks:
 - name: restic.admission.stash.appscode.com
   clientConfig:
     service:
-      namespace: default
+      namespace: {{ .Release.Namespace }}
```

Comment:
All changes made in this file are incorrect. The webhook server is registered as an extended apiserver and only accessed through the default/kubernetes service.

Reply:
ok, will revert